File: kevoree-library/javase/org.kevoree.library.javase.webSocket/src/main/java/org/kevoree/library/ws/Tester.java
package org.kevoree.library.ws;

import net.tootallnate.websocket.WebSocket;

import java.net.URI;
import java.net.UnknownHostException;

/**
 * Created by IntelliJ IDEA.
 * User: duke
 * Date: 14/02/12
 * Time: 21:32
 * To change this template use File | Settings | File Templates.
 */
public class Tester  {

    public static void main(String[] args){

        try {
         //   WebSocket.DEBUG = true;
           /// WsServer ws = new WsServer();
          //  ws.start();
            
            System.out.println("Server Started !");

            WsSocketClient client = new WsSocketClient(new URI("ws://localhost:9090"));
            client.connect();



        } catch (Exception e) {
            e.printStackTrace();
        }


    }


}

File: kevoree-corelibrary/javase/org.kevoree.library.javase.webserver.servlet/src/main/scala/org/kevoree/library/javase/webserver/servlet/Tester.scala
package org.kevoree.library.javase.webserver.servlet

import javax.servlet.http.HttpServlet
import org.kevoree.library.javase.webserver.{KevoreeHttpResponse, KevoreeHttpRequest}

/**
 * Created by IntelliJ IDEA.
 * User: duke
 * Date: 20/12/11
 * Time: 08:12
 * To change this template use File | Settings | File Templates.
 */

object Tester extends App {

  println("Hi")


  val lr = new LocalServletRegistry

  lr.registerServlet("/*",new HttpServlet{})
//  println(lr.tryURL("/",new KevoreeHttpRequest,new KevoreeHttpResponse))



}